entity-framework/core/managing-schemas/migrations/index.md

```diff
@@ -125,12 +152,11 @@ Remove-Migration
 dotnet ef migrations remove
 ```
 
-After removing it, you can make the additional model changes and add it again.
+After removing the migration, you can make the additional model changes and add it again.
```

Comment:
Add another sentence explaining what it does?

> … This both reverts the database schema changes from the latest migration applied and remove that migration from the application's project.

Reply:
:blush: IIRC we discussed a change for that in some version. @bricelam would know.

Reply:
@tdykstra yeah, you were right. That is more less what we implemented in 2.1: https://github.com/aspnet/EntityFrameworkCore/issues/10781#issuecomment-362414534